remotedcurl/remotedcurl.c

```diff
@@ -0,0 +1,135 @@
+#include <assert.h>
+
+#include "amqp.h"
+#include "amqp_tcp_socket.h"
+
+#include "constants.h"
+#include "dcurl.h"
+
+#define HOSTNAME "localhost"
+#define PORT 5672
+#define HEATBEATS 0  // disables heartbeats
```

Comment:
Why?

Reply:
In my VM test environment, if heartbeat sets, the broker misses heartbeats from dcurl work, default timeout is 1s. Error message is as follows: 
```
rabbitmq_1  | missed heartbeats from client, timeout: 1s
```

Reply:
Check the manual of RabbitMQ  in explanation of specific configurations such as `HEATBEATS`.